spec/hanami/view_spec.rb

```diff
@@ -381,19 +381,17 @@ class ConfigurationLayout
       end
 
       it 'a view inherits the configuration from the framework' do
-        pending('Does == mean something different in rspec and minitest?')
         expected = Hanami::View.configuration
         actual   = ConfigurationView.configuration
 
-        expect(actual).to eq(expected)
+        expect(actual.root).to eq(expected.root)
```

Comment:
@nerdinand This doesn't consider all the other settings in the configuration.

Here we're exercising `Configuration#==`, so it's worth to make this assertion explicit, instead of relying on the test framework implicit comparison. This should work:

```ruby
expect(actual == expected).to be(true)
```